SUMMARY.md

```diff
@@ -29,12 +29,13 @@
     * [Payment card authorization](booking-engine-api/use-cases/payment-card-authorization.md)
     * [Supporting payment cards](booking-engine-api/use-cases/supporting-payment-cards.md)
   * [API Operations](booking-engine-api/operations/README.md)
-	  * [Configuration](booking-engine-api/operations/configuration.md)
-	  * [Hotels](booking-engine-api/operations/hotels.md)
-	  * [Payment cards](booking-engine-api/operations/payment-cards.md)
-	  * [Reservations](booking-engine-api/operations/reservations.md)
-	  * [Reservation groups](booking-engine-api/operations/reservation-groups.md)
-	  * [Vouchers](booking-engine-api/operations/vouchers.md)
+      * [Configuration](booking-engine-api/operations/configuration.md)
+      * [Hotels](booking-engine-api/operations/hotels.md)
+      * [Availability blocks](booking-engine-api/operations/availability-blocks.md)
```

Comment:
1. Can we move the new entry for Availability blocks in Operations up to the top please, because they are listed in alphabetical order?

2. We also need a new entry in SUMMARY.md for the new use cases page for Availability blocks.